src/components/mag-forces-panel.tsx

```diff
@@ -0,0 +1,48 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./base";
+
+import "./mag-forces-panel.sass";
+import { SwitchComponent } from "./switch";
+
+interface IProps extends IBaseProps {}
+interface IState {}
+
+@inject("stores")
+@observer
+export class MagForcesPanelComponent extends BaseComponent<IProps, IState> {
+
+  public render() {
+    const {simulation} = this.stores;
+    const forcesLabel: string = simulation.showMagneticForces ? "ON" : "OFF";
+    const forcesOn: boolean = forcesLabel === "ON" ? true : false;
```

Comment:
Could probably trim this variable, looks like it's equivalent to `simulation.showMagneticForces`